style.css

```diff
@@ -0,0 +1,32 @@
+* {
+ margin: 0;
+ padding: 0;
+ width: 100%;
```

Comment:
generally not a good idea to set the * width to 100%, you will most likely get odd behaviour out of it.